### PR TITLE
feat: add information in regards to release forms

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says. Pinging involved parties for review

@jorgepiloto @MaxJPRey @ChrisHarrold

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]
Thanks for adding this, @RobPasMue. I hope it helps to clarify our OSS process for the different teams.

File: doc/source/getting-started/administration.rst [hunk 1]
```diff
@@ -4,6 +4,16 @@ Most of the projects in PyAnsys expose the functionality of Ansys
 products. Due to intellectual property reasons, the public release of a PyAnsys
 library must go through a project approval process.
 
+First steps
+-----------
+In order to trigger the public release process, project leads should fill in the following
+form:
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:10]:
```suggestion
To trigger the public release process, project leads must fill in this form:
```

File: doc/source/getting-started/administration.rst [hunk 2]
```diff
@@ -4,6 +4,16 @@ Most of the projects in PyAnsys expose the functionality of Ansys
...
+
+* `Release request workflow initiation form <https://gz2idtcjsw2.typeform.com/to/qVujtwSR>`_
+
+This form lets the different parties involved in the public release process know that
+there is a request to release a project. If your selected option was to release an Ansys Open
+Source project, then please continue to the next section.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:16]:
```suggestion
The form lets the different parties involved in the public release process know that
there is a request to release a project. If your intent is to release an Ansys Open
Source project, then continue to the next section.
```

File: doc/source/getting-started/administration.rst [hunk 3]
```diff
@@ -46,6 +56,15 @@ When releasing a project to the public, you should:
 * Uphold Ansys' reputation in the open source community.
 
 
+Once all these three approvals have been awarded, project leads are requested to fill
+in the next form:
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:60]:
```suggestion
Once all three approvals have been awarded, project leads must complete
the next form:
```

File: doc/source/getting-started/administration.rst [hunk 4]
```diff
@@ -46,6 +56,15 @@ When releasing a project to the public, you should:
...
+
+* `OSS approval request form <https://gz2idtcjsw2.typeform.com/OSSapproval>`_
+
+This form serves as a final checklist to verify that all approvals have been processed
+and to formalize the OSS approval process, as a final record. You can find the questions
+to be asked in :ref:`OSS approval request form questions`.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:66]:
```suggestion
This form serves as a final checklist to verify that all approvals have been processed
and to formalize the OSS approval process as a final record. You can find frequently
asked questions in :ref:`OSS approval request form questions`.
```

File: doc/source/getting-started/administration.rst [hunk 5]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
 
     <input disabled="" type="checkbox">
+
+OSS approval request form questions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When filling in the `OSS approval request form`_, project leads have to
+answer the following questions (they might be of different nature: fill in,
+yes/no, or multiple-choice). Be ready to answer them once you start completing
+the form.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:194]:
```suggestion
When filling in the `OSS approval request form`_, project leads must
supply responses to these questions:.
```
Does it matter what the format of questions on the form are?

File: doc/source/getting-started/administration.rst [hunk 6]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * Who is the ACE/AFT owner?
+
+.. card:: |uncheck| Legal questions
+
+    * Who validated your legal readiness?
+    * Provided there are no issues with the MIT license, have you correctly applied
+      it to the GitHub Repository for your project?
+    * Is the copyright header correctly applied to your files in GitHub?
+    * Have you confirmed that any IP is removed from the code, docs, and examples?
+    * I and my legal reviewer as well as my product and PM reviewer have confirmed that
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:211]:
```suggestion
    * I and my legal reviewer, as well as my product and PM reviewer, have confirmed that
```

File: doc/source/getting-started/administration.rst [hunk 7]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+      there is no business interest in keeping this code confidential.
+    * I and my legal reviewer confirm there is no business interest in enforcing copyright
+      protection for this code.
+    * I and my legal reviewer confirm that the code does not contain any 3rd party material
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:215]:
```suggestion
    * I and my legal reviewer confirm that the code does not contain any third-party material
```

File: doc/source/getting-started/administration.rst [hunk 8]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+      (open source, proprietary, partner, customer, or otherwise).
+    * I and my legal reviewer confirm that the code does not include any invention on which
+      the company has, or might want to seek a patent.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:218]:
```suggestion
      the company has, or might want to seek, a patent.
```

File: doc/source/getting-started/administration.rst [hunk 9]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * Have you cleaned up the comments/issues/PRs/etc. to remove any potentially bad things?
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:219]:
```suggestion
    * Have you cleaned up comments, issues, and pull requests to remove any potentially bad content?
```

File: doc/source/getting-started/administration.rst [hunk 10]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * My legal reviewer and I have checked the dependencies and validated they do not
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:220]:
```suggestion
    * My legal reviewer and I have checked the dependencies and validated that they do not
```

File: doc/source/getting-started/administration.rst [hunk 11]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+      impose any licensing difficulties?
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:221]:
```suggestion
      impose any licensing difficulties.
```

File: doc/source/getting-started/administration.rst [hunk 12]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * I and my legal reviewer confirm there is NO encryption present in the code.
+    * The repository that hosts the code is generally accessible to the public with no
+      time limits or access restrictions.
+    * This tool or library is not meant for use in any specific industry, platform, or
+      process and is meant for use by general customers.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:226]:
```suggestion
      process but rather for use by general customers.
```

File: doc/source/getting-started/administration.rst [hunk 13]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+
+.. card:: |uncheck| Technical questions
+
+    * Who verified your technical review?
+    * Was your documentation for usage/installation/pre-reqs reviewed by the docs team?
+    * Has your source code documentation been reviewed?
+    * Completed end user testing?
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:233]:
```suggestion
    * Has end user testing been completed?
```

File: doc/source/getting-started/administration.rst [hunk 14]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * Pull request testing complete?
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:234]:
```suggestion
    * Has pull request testing been completed?
```
What does this even mean?

File: doc/source/getting-started/administration.rst [hunk 15]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+    * Minimum of 80% testing coverage verified?
+    * Your usage and installation examples are ready?
+    * Package definition ready and PyPi packaging completed?
+    * GitHub Repository has the contribution guidance and CLA set up and ready.
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:238]:
```suggestion
    * Has a minimum test coverage of 80% been achieved?
    * Are usage and installation examples included and tested?
    * Is the package definition ready and PyPi packaging completed?
    * Does the GitHub repository supply contribution guidance and have CLA set up?
```

File: doc/source/getting-started/administration.rst [hunk 16]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+
+.. card:: |uncheck| Business questions
+
+    * Who did you get from the PMM/DevEco team to check your readiness?
+    * Did you tell ACE and the Business Unit you are at this point?
+    * Is there something public that is already called the same thing you want to call your project?
+    * Did you get Product Marketing Manager (PMM) signoff?
+    * Did you let DevEco know so they can update links from the DevPortal to your new OSS project?
+    * Did you let Product Management know that your library is nearing release?
```
Comment (PipKat) [inline doc/source/getting-started/administration.rst:247]:
```suggestion
    * Who on the Product Marketing Manager (PMM) or Developer Ecosystem (DevEco)
      team checked your project for readiness?
    * Did you tell ACE and your Business Unit lead that you are ready for release?
    * Is there something public that already has the same name as your project?
    * Did you get PMM signoff?
    * Did you ask the DevEco team to update links from the Developer Portal to your new OSS project?
    * Did you let the PMM team know that your library is nearing release?
```

File: doc/source/getting-started/administration.rst [hunk 17]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
...
+yes/no, or multiple-choice). Be ready to answer them once you start completing
+the form.
+
+.. card:: |uncheck| General questions
+
+    * What is the name of your project?
+    * Who is the project maintainer?
+    * Who is the lead from the product team?
+    * Who is the Product Management contact?
+    * Who is the ACE/AFT owner?
```
[the remark this answers is not in the document]
Comment (RobPasMue) [reply doc/source/getting-started/administration.rst:199]: Yep, let's do that. =)

[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/getting-started/administration.rst:199]: @phxnsharp and @RobPasMue I don't think that Sharepoint page is public.

[the remark this answers is not in the document]
Comment (phxnsharp) [reply doc/source/getting-started/administration.rst:199]: Sorry, I didn't realize this documentation was public. The SECOE website is intentionally "internal only" to Ansys to allow internal Ansys people to freely add content without worrying about legal or marketing concerns. Good catch @PipKat.

File: doc/source/getting-started/administration.rst [hunk 18]
```diff
@@ -165,3 +184,64 @@ The technical review of the project verifies the following:
 .. |uncheck_| raw:: html
 
     <input disabled="" type="checkbox">
+
+OSS approval request form questions
```
[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/getting-started/administration.rst:187]: @greschd So true! I made an assumption and forgot to go back to the section title to edit after I understood the content. Thank you!

**Comment by PipKat** [comment]
> Let me get back to all your points in my next commits @greschd @PipKat @phxnsharp
> 
> I will rename the title and I will try to address the acronyms issue (somehow)

@RobPasMue If you want to ceate an issue and assign it to me, I will comb through the dev guide and collect abbreviations and acronyms in User guide topic.

**Comment by RobPasMue** [comment]
I just solved the title issue. I'll leave the acronyms to @PipKat since she offered to help (really appreciated @PipKat as always! 😄)

Merging! Thank you all for your feedback!

**Pull request merged**